integrations/langfuse/src/haystack_integrations/components/connectors/langfuse/langfuse_connector.py

```diff
@@ -112,6 +114,7 @@ def __init__(
             only accessible to the Langfuse account owner. The default is `False`.
         :param public_key: The Langfuse public key. Defaults to reading from LANGFUSE_PUBLIC_KEY environment variable.
         :param secret_key: The Langfuse secret key. Defaults to reading from LANGFUSE_SECRET_KEY environment variable.
+        :param httpx_client: Optional custom httpx.Client instance to use for Langfuse API calls.
```

Comment:
Perhaps we should mention the serialization limitation in the docstring.

Reply:
Very good point. Have one last quick look again 🙏